app/controllers/admin_controller.rb

```diff
@@ -2,10 +2,27 @@
 class AdminController < ApplicationController
   before_action :authenticate_request!
   before_action :current_user_admin
+
   def index
     render json: { 'logged_in' => true }
   end
 
+  def index_admin
```

Comment:
@domenic99 maybe, it's better to use action `index` for this, to be relevant to resource routing?